metagraph/src/common/serialization.cpp

```diff
@@ -106,6 +181,117 @@ template bool load_number_vector<uint8_t>(std::istream &in, std::vector<uint8_t>
 template bool load_number_vector<bool>(std::istream &in, std::vector<bool> *);
 
 
+void encode_utf8(uint32_t num, std::ostream &out) {
```

Comment:
Could you add that statement "adapted from libmaus2 for backward compatibility" here as well?

Reply:
I ended up only keeping `decode_utf8` from libmaus2, everything else has been implemented differently.